package.json

```diff
@@ -1,11 +1,12 @@
 {
   "devDependencies": {
-    "@semantic-release/changelog": "^6.0.3",
-    "@semantic-release/release-notes-generator": "^14.0.1",
+    "@semantic-release/changelog": "^5.0.1",
```

Comment:
why are you downgrading versions?

Reply:
Not sure how this happened, could be due to pyright problems i was facing, reverted